tools/pot/openvino/tools/pot/engines/ie_engine.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, config, data_loader=None, metric=None):
         self._per_sample_metrics = []
         self._tmp_dir = create_tmp_dir()
         self._device = self.config.device
+        self.calculate_metrics = False
```

Comment:
Can it corrupt the behavior in the simplified mode execution? I mean, metrics calculation during execution using POT SW API. As I know, our API doesn't provide any variable setters or an example, of how to enable or disable metrics calculation.

Reply:
In Simplified we don't compute metrics. As I can see, AA method does the following `self._engine.calculate_metrics = True`. But it makes sense for reporting a metric when IEEngine is used as a helper in the user's implementation of quantization flow.

Reply:
`SimplifiedEngine` inherits from `IEEngine`, but it doesn't matter. It's just what I'm talking about: metric reporting will be disabled as default and, maybe, we should highlight this or implement this variable as property?